train/tr10-13B-ml/tr10-13B.slurm

```diff
@@ -57,13 +57,14 @@ OPTIMIZER_ARGS=" \
     --adam-beta1 0.9 \
     --adam-beta2 0.95 \
     --adam-eps 1e-8 \
-    --lr 6e-5 \
+    --lr 1e-4 \
```

Comment:
GPT3 paper suggest a higher learning rate. Is there a reason why we would use `6e-5`?